css/alumni.sass

```diff
@@ -5,15 +5,31 @@
 #collegeName 
     font-size: 30px
     color: black
+
 #collegeImg 
     max-width: 150px
     max-height: auto
     display: block
     margin-left: auto
     margin-right: auto
- 
-    
+
+li.quotes
+    list-style: none
+    list-style-type: none
```

Comment:
Why even use a list if we're removing all the style? How about just plain blockquotes?

Reply:
I tried using just blockquotes, but that removes indentation

Reply:
```sass
blockquote
    margin-left: 40px
```

Reply:
```sass
blockquote
    margin-left: 40px
```

Reply:
I by switched up my words there, but that works. The <li>s are gone